commit 6664f9e7a1e69e929a0aa5214920558d3076c27e
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Fri Mar 8 11:26:00 2019 +0000

    staging: bcm2835_codec: Clean up logging on unloading the driver
    
    The log line was missing a closing \n, so wasn't added to the
    log immediately.
    Adds the function of the V4L2 device that is being unregistered
    too.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c b/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
index ad555ceb8f63..837fbec5ab4f 100644
--- a/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
+++ b/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
@@ -77,6 +77,12 @@ enum bcm2835_codec_role {
 	ISP,
 };
 
+const static char *roles[] = {
+	"decode",
+	"encode",
+	"isp"
+};
+
 static const char * const components[] = {
 	"ril.video_decode",
 	"ril.video_encode",
@@ -2522,7 +2528,6 @@ static int bcm2835_codec_create(struct platform_device *pdev,
 	struct video_device *vfd;
 	int video_nr;
 	int ret;
-	const static char *roles[] = {"decode", "encode", "isp"};
 
 	dev = devm_kzalloc(&pdev->dev, sizeof(*dev), GFP_KERNEL);
 	if (!dev)
@@ -2615,7 +2620,8 @@ static int bcm2835_codec_destroy(struct bcm2835_codec_dev *dev)
 	if (!dev)
 		return -ENODEV;
 
-	v4l2_info(&dev->v4l2_dev, "Removing " MEM2MEM_NAME);
+	v4l2_info(&dev->v4l2_dev, "Removing " MEM2MEM_NAME ", %s\n",
+		  roles[dev->role]);
 	v4l2_m2m_release(dev->m2m_dev);
 	video_unregister_device(&dev->vfd);
 	v4l2_device_unregister(&dev->v4l2_dev);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1531_6664f9e7a1e69e929a0aa5214920558d3076c27e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1531_6664f9e7a1e69e929a0aa5214920558d3076c27e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
